Editor/Window/LandingPage.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using AmazonGameLift.Editor;
+using Editor.Resources.EditorWindow;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace Editor.Window
+{
+    public class LandingPage
+    {
+        private readonly VisualElement _container;
+
+        public LandingPage(VisualElement container)
+        {
+            _container = container;
+            var mVisualTreeAsset = UnityEngine.Resources.Load<VisualTreeAsset>("EditorWindow/Pages/LandingPage");
+            var uxml = mVisualTreeAsset.Instantiate();
+            
+            container.Add(uxml);
+            ApplyText();
```

Comment:
`Apply` doesn't really mean anything in this situation. We should be more specific with our method naming, something like `LocalizeText` or `LocalizeStrings`

Reply:
The same should apply to GameLiftPlugin and future pages

Reply:
renamed!